install/install.queries.php

```diff
@@ -704,14 +687,14 @@ function encryptFollowingDefuse($message, $ascii_key)
                             array('admin', 'pwd_default_length', '14'),
                             array('admin', 'tasks_log_retention_delay', '30'),
                             array('admin', 'oauth2_enabled', '0'),
-                            array('admin', 'oauth2_client_id', ''),
-                            array('admin', 'oauth2_client_secret', ''),
+                            array('admin', 'oauth2_client_id', '', '1'),
+                            array('admin', 'oauth2_client_secret', '', '1'),
                             array('admin', 'oauth2_client_endpoint', ''),
-                            array('admin', 'oauth2_client_token', ''),
+                            array('admin', 'oauth2_client_token', '', '1'),
```

Comment:
The last `'1'` is for encryption?
`oauth2_client_token` contains the URL of tokeninfo endpoint and not a confidential token.

Reply:
Yes but as tpconfig file has been removed, it is not used anymore.
I will merge the PR and remove this after.
OK for you?